packages/cozy-doctypes/src/Document.js

```diff
@@ -284,6 +284,50 @@ class Document {
 
     return { newLastSeq, documents: docs }
   }
+
+  /**
+   * Fetches all documents for a given doctype exceeding the 100 limit.
+   * It is slower that fetchAll because it fetches the data 100 by 100 but allows to filter the data
+   * with a selector and an index
+   *
+   * Parameters:
+   *
+   * * `doctype` (string): the doctype from which you want to fetch the data
+   * * `selector` (object): the mango query selector
+   * * `index` (object): (optional) the query selector index. If not defined, the function will
+   * create it's own index with the keys specified in the selector
+   *
+   *
+   * ```javascript
+   * const documents = await queryAll(cozyClient, 'io.cozy.bills', {vendor: 'Direct Energie'})
+   * ```
+   *
+   */
+  static async queryAll(selector, index) {
```

Comment:
In the doc we have 3 parameters but the function takes only 2 args

Reply:
Thanks !